subsys/pm/device_runtime.c

```diff
@@ -201,6 +203,7 @@ void pm_device_runtime_enable(const struct device *dev)
 
 end:
 	SYS_PORT_TRACING_FUNC_EXIT(pm, device_runtime_enable, dev);
```

Comment:
nit: SYS_PORT_TRACING_FUNC_EXIT should also be updated to collect return code (and other related definitions such as `sys_port_trace_pm_device_runtime_enable_exit`).